Assets/HoloToolkit/UX/Scripts/Collections/CollectionEnums/OrientTypeEnum.cs

```diff
@@ -11,7 +11,11 @@ public enum OrientTypeEnum
         None,                   // Don't rotate at all
         FaceOrigin,             // Rotate towards the origin
         FaceOriginReversed,     // Rotate towards the origin + 180 degrees
-        FaceFoward,             // Zero rotation
-        FaceForwardReversed,    // Zero rotation + 180 degrees
+        FaceParentFoward,       // Parent Relative Forwards
+        FaceParentBack,         // Parent Relative Backwards
+		FaceParentUp,           // Parent Relative Up
```

Comment:
strange formatting

Reply:
This extra tab doesn't show up in my local copy. Is this a merge artifact that I can fix on github?

Reply:
It looks like it's because `FaceParentUp` is preceded with tabs, while the others are spaces.

Reply:
Aha! Thank you!